### PR TITLE
Add PathSeq WDL

### PR DESCRIPTION
**Pull request by mwalker174**
WDL for running PathSeq pipeline with a readme and example json input.

### CODE REVIEW

File: scripts/pathseq/wdl/pathseq_pipeline.wdl [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    Boolean? skip_pre_bwa_repartition
+    Boolean? divide_by_genome_length
+    Int? filter_bwa_seed_length
+    Int? host_min_identity
+    Int? min_clipped_read_length
+    Int? bam_partition_size
+    Float? min_score_identity
+    Float? identity_margin
+
+	File? gatk4_jar_override
```
Comment (LeeTL1220) [inline scripts/pathseq/wdl/pathseq_pipeline.wdl]: Can you de-dent to make these line up with the parameters above?

File: scripts/pathseq/wdl/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+PathSeq reference files are available in the [GATK Resource Bundle](https://software.broadinstitute.org/gatk/download/bundle) (direct link [here](ftp://ftp.broadinstitute.org/bundle/beta/PathSeq)).
+
+*Please note that there are optional parameters that do not appear in the template files, since we do not want to specify, by default*
+
+### Docker image
+- "broadinstitute/gatk:4.0.0.0"
+
+### Parameter descriptions
+
+Recommended default values (where possible) are found in ``pathseq_pipeline_template.json``
```
[the remark this answers is not in the document]
Comment (mwalker174) [reply scripts/pathseq/wdl/README.md]: Added optional parameters and indicates which resource bundle files map to which parameters.

File: scripts/pathseq/wdl/pathseq_pipeline.wdl (hunk 1 above)
Comment (mwalker174) [reply scripts/pathseq/wdl/pathseq_pipeline.wdl]: Cleaned up indentation (double-spaces everywhere)

File: scripts/pathseq/wdl/pathseq_pipeline.wdl [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    # Some tasks need wiggle room, and we also need to add a small amount of disk to prevent getting a
+    # Cromwell error from asking for 0 disk when the input is less than 1GB.
+    # Also Spark requires some temporary storage.
+    Int additional_disk = select_first([increase_disk_size, 20])
+
+    # Disk sizes for Downsample and PathSeq tasks
+    Float disk_space_gb = size(input_bam, "GB") + size(kmer_file, "GB") + size(filter_bwa_image, "GB") + size(microbe_bwa_image, "GB") + size(microbe_fasta, "GB") + additional_disk
+
+    call PathseqPipeline {
+    input:
```
[the remark this answers is not in the document]
Comment (mwalker174) [reply scripts/pathseq/wdl/pathseq_pipeline.wdl]: Done

File: scripts/pathseq/wdl/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+- ``PathSeqPipelineWorkflow.identity_margin`` -- If a read maps to more than one microbe, the best alignment is counted. Any additional alignments are also counted if they are within this margin of the best alignment. 
+- ``PathSeqPipelineWorkflow.divide_by_genome_length`` -- If true, abundance scores are normalized by genome length.
+- ``PathSeqPipelineWorkflow.filter_bwa_image`` -- Path to host BWA index image. This corresponds to `pathseq_host.fa.img` in the Resource Bundle.
+- ``PathSeqPipelineWorkflow.kmer_file`` -- Path to host k-mer file.  This corresponds to `pathseq_host.bfi` in the Resource Bundle.
+- ``PathSeqPipelineWorkflow.microbe_bwa_image`` -- Path to microbe BWA index image. This corresponds to `pathseq_microbe.fa.img` in the Resource Bundle.
+- ``PathSeqPipelineWorkflow.microbe_fasta`` -- Path to microbe reference FASTA file. This corresponds to `pathseq_microbe.fa` in the Resource Bundle.
+- ``PathSeqPipelineWorkflow.microbe_fasta_dict`` -- Path to microbe reference dictionary file.This corresponds to `pathseq_microbe.dict` in the Resource Bundle.
```
Comment (LeeTL1220) [inline scripts/pathseq/wdl/README.md]: `.  This`

File: scripts/pathseq/wdl/README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+- ``PathSeqPipelineWorkflow.taxonomy_file`` -- Path to PathSeq taxonomy file. This corresponds to `pathseq_taxonomy.db` in the Resource Bundle.
+- ``PathSeqPipelineWorkflow.gatk_docker`` -- GATK docker image
+
+Optional parameters:
+
+- ``PathSeqPipelineWorkflow.skip_quality_filters`` -- If true, skips quality filtering steps
```
Comment (LeeTL1220) [inline scripts/pathseq/wdl/README.md]: A couple of these are missing default values.  Can you populate?

File: scripts/pathseq/wdl/README.md (hunk 4 above)
Comment (mwalker174) [reply scripts/pathseq/wdl/README.md]: Done

File: scripts/pathseq/wdl/README.md (hunk 5 above)
Comment (mwalker174) [reply scripts/pathseq/wdl/README.md]: Added default value for all optional parameters. I also realized some of the optional parameters were in the first section so I moved them down.

**Pull request merged**